mitodl/mitxonline
# 906 color contrast issues on dashboard

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/906

#### What's this PR do?
Updating the color contrast for red, grey, and green to be at least 4.5 on the dashboard
- Red is changed to #B43E3E / #fae8e8 (contrast ratio 4.8)
- Green is changed to #3E775D / #DBF3E9 (contrast ratio 4.5)
- Gray is changed to #666666 / white (contrast ratio 5.7)

#### How should this be manually tested?
Go to dashboard to check the color
![image](https://user-images.githubusercontent.com/3138890/187500053-da815741-5324-4181-ada4-5b6b031f2da4.png)
